src/js/plugins/base.js

```diff
@@ -11,7 +11,8 @@ ripe.Ripe.plugins.Plugin = function() {
     ripe.Observable.call(this);
 };
 
-ripe.assign(ripe.Ripe.plugins.Plugin.prototype, ripe.Observable.prototype);
+ripe.Ripe.plugins.Plugin.prototype = Object.create(ripe.Observable.prototype);

```

Comment:
why this change ? Bug fix ?

Reply:
It's the same approach used on the plugins